src/app/modules/gb-analysis-module/fractalis-components/gb-fractalis-visual/gb-fractalis-visual.component.html

```diff
@@ -4,16 +4,27 @@
     <button type="button" class="btn btn-primary btn-sm" (click)="onClearCharts()">Clear</button>
   </div>
 
-  <gridster [options]="options">
-    <gridster-item [item]="chart.gridsterItem" *ngFor="let chart of charts">
-      <div>
-        <span class="close-btn-row float-right">
-          <button mat-icon-button color="primary" (click)="onRemoveChart($event, chart)">
-            <mat-icon aria-label="Close the chart">close</mat-icon>
-          </button>
-        </span>
+  <div class="charts">
+    <div *ngFor="let chart of charts">
+      <span [ngClass]="{'fractatis-chart-hidden': !chart.isValid}">
+        <button mat-icon-button color="primary" (click)="onRemoveChart($event, chart)">
+          <mat-icon aria-label="Close the chart">close</mat-icon>
+        </button>
         <gb-fractalis-chart [chart]="chart"></gb-fractalis-chart>
-      </div>
-    </gridster-item>
-  </gridster>
+      </span>
+    </div>
+  </div>
+
+  <!--<gridster [options]="options">-->
```

Comment:
can remove this

Reply:
yes, it will be a part of https://jira.thehyve.nl/browse/TMT-628 (removing gridster code is part of the ticket). I kept this, because previously TMT-628 was to either fix gridster, or introduce some other grid layout library. It was clarified recently.